libnavigation-ui/src/main/java/com/mapbox/navigation/ui/NavigationViewOptions.java

```diff
@@ -170,9 +172,9 @@
     public abstract NavigationViewOptions build();
   }
 
-  public static Builder builder() {
+  public static Builder builder(Context context) {
     return new AutoValue_NavigationViewOptions.Builder()
-        .navigationOptions(new NavigationOptions.Builder().build())
+        .navigationOptions(new NavigationOptions.Builder(context).build())
```

Comment:
This looks like an issue in master as well. Does overriding NavigationOptions even work?